2025-07-01 10:25:50,208 INFO    MainThread:4880 [wandb_setup.py:_flush():81] Current SDK version is 0.20.1
2025-07-01 10:25:50,208 INFO    MainThread:4880 [wandb_setup.py:_flush():81] Configure stats pid to 4880
2025-07-01 10:25:50,208 INFO    MainThread:4880 [wandb_setup.py:_flush():81] Loading settings from C:\Users\dgeri\.config\wandb\settings
2025-07-01 10:25:50,208 INFO    MainThread:4880 [wandb_setup.py:_flush():81] Loading settings from D:\INACAP\2024 Ingenieria\8vo semestre\Machine Learning\Unidad 4\DLOps\wandb\settings
2025-07-01 10:25:50,208 INFO    MainThread:4880 [wandb_setup.py:_flush():81] Loading settings from environment variables
2025-07-01 10:25:50,208 INFO    MainThread:4880 [wandb_init.py:setup_run_log_directory():703] Logging user logs to .\wandb\run-20250701_102550-l3exykhr\logs\debug.log
2025-07-01 10:25:50,209 INFO    MainThread:4880 [wandb_init.py:setup_run_log_directory():704] Logging internal logs to .\wandb\run-20250701_102550-l3exykhr\logs\debug-internal.log
2025-07-01 10:25:50,209 INFO    MainThread:4880 [wandb_init.py:init():831] calling init triggers
2025-07-01 10:25:50,209 INFO    MainThread:4880 [wandb_init.py:init():836] wandb.init called with sweep_config: {}
config: {'datamodule': {'path': WindowsPath('dataset'), 'batch_size': 25}, 'trainer': {'max_epochs': 10, 'enable_checkpointing': True, 'overfit_batches': 0, 'logger': <pytorch_lightning.loggers.wandb.WandbLogger object at 0x00000210D6F9F080>, 'callbacks': [<src.utils.WandBCallback object at 0x00000210D63E9CA0>, <pytorch_lightning.callbacks.model_checkpoint.ModelCheckpoint object at 0x00000210D71FAC30>, <pytorch_lightning.callbacks.progress.tqdm_progress.TQDMProgressBar object at 0x00000210D72209B0>, <pytorch_lightning.callbacks.model_summary.ModelSummary object at 0x00000210D7220B30>]}, 'logger': 'WandbLogger', 'callbacks': [{'name': 'WandBCallback', 'lib': 'src.utils', 'params': {'labels': ['no 3', '3'], 'dl': <torch.utils.data.dataloader.DataLoader object at 0x00000210D71FAC00>}}, {'name': 'ModelCheckpoint', 'lib': 'pytorch_lightning.callbacks', 'params': {'dirpath': 'checkpoints', 'filename': '006-{val_loss:.5f}-{epoch}', 'save_top_k': 1, 'monitor': 'val_loss', 'mode': 'min'}}], 'logger_params': {'project': 'dlops-mnist', 'name': '006'}, '_wandb': {}}
2025-07-01 10:25:50,209 INFO    MainThread:4880 [wandb_init.py:init():872] starting backend
2025-07-01 10:25:50,429 INFO    MainThread:4880 [wandb_init.py:init():875] sending inform_init request
2025-07-01 10:25:50,482 INFO    MainThread:4880 [wandb_init.py:init():883] backend started and connected
2025-07-01 10:25:50,483 INFO    MainThread:4880 [wandb_init.py:init():956] updated telemetry
2025-07-01 10:25:50,546 INFO    MainThread:4880 [wandb_init.py:init():980] communicating run to backend with 90.0 second timeout
2025-07-01 10:25:51,484 INFO    MainThread:4880 [wandb_init.py:init():1032] starting run threads in backend
2025-07-01 10:25:51,703 INFO    MainThread:4880 [wandb_run.py:_console_start():2453] atexit reg
2025-07-01 10:25:51,703 INFO    MainThread:4880 [wandb_run.py:_redirect():2301] redirect: wrap_raw
2025-07-01 10:25:51,703 INFO    MainThread:4880 [wandb_run.py:_redirect():2370] Wrapping output streams.
2025-07-01 10:25:51,703 INFO    MainThread:4880 [wandb_run.py:_redirect():2393] Redirects installed.
2025-07-01 10:25:51,707 INFO    MainThread:4880 [wandb_init.py:init():1078] run started, returning control to user process
2025-07-01 10:25:52,702 INFO    MainThread:4880 [wandb_run.py:_config_callback():1358] config_cb None None {'datamodule': {'path': 'dataset', 'batch_size': 25}, 'trainer': {'max_epochs': 10, 'enable_checkpointing': True, 'overfit_batches': 0}, 'logger': 'WandbLogger', 'callbacks': [{'name': 'WandBCallback', 'lib': 'src.utils', 'params': {'labels': ['no 3', '3']}}, {'name': 'ModelCheckpoint', 'lib': 'pytorch_lightning.callbacks', 'params': {'dirpath': 'checkpoints', 'filename': '006', 'save_top_k': 1, 'monitor': 'val_loss', 'mode': 'min'}}], 'logger_params': {'project': 'dlops-mnist', 'name': '006'}}
2025-07-01 10:26:06,726 INFO    MsgRouterThr:4880 [mailbox.py:close():129] [no run ID] Closing mailbox, abandoning 2 handles.
